assignments/arrays.js

```diff
@@ -63,32 +63,57 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+for(let i = 0; i < inventory.length ; i++){
+    if (i === 33){
+        console.log('Car 33 is a ' + inventory[32]['car_year'] + ' ' + inventory[32]['car_make'] + ' ' + inventory[32]['car_model'] );
+    }
+}
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+ lastcar = inventory.slice(-1)[0]
```

Comment:
```suggestion
let lastcar = inventory.slice(-1)[0]
```
Cool use of this. You could have also accessed the last item by using inventory[inventory.length - 1] . We know the last index of an array is going to be one less than the array's length.